content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,179 @@
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+   .. image:: in_app_purchase/sms-icon.png
+      :align: center
+      :alt: The SMS icon on a typical contact information form located within an Odoo database.
 
-.. image:: in_app_purchase/image4.png
-   :align: center
+   One way to utilize the *SMS* IAP service with Odoo is showcased in the following steps:
+
+   1. Navigate to the *Contacts* application, and click on a contact with a mobile phone number
+      in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the contact form.
+   2. Hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field, and a :guilabel:`📱
+      (phone) SMS` icon appears to the right.
+   3. Click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text Message` pop-up
+      window appears.
+   4. Type a message in the :guilabel:`Message` field of the :guilabel:`Send SMS Text Message`
+      pop-up window. Then, click the :guilabel:`Send SMS` button, and Odoo sends the message, via
+      SMS, to contact, and logs what was sent in the *chatter* of the contact's form.
+
+   Upon sending the SMS message, the prepaid credits for the *SMS* IAP service are automatically
+   deducted from the existing credits. If there are not enough credits to send the message, Odoo
+   prompts the user to purchase of more.
+
+.. seealso::
+   For more information on how to use various IAP services, and for more in-depth instructions
+   related to SMS functionality in Odoo, review the documentation below:
+
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an IAP service is used, the prepaid credits for that service are spent. Odoo prompts the
+purchase of more credits when there are not enough credits to continue using the service. Email
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in *Packs* from the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - :guilabel:`Starter Pack`: 10 credits
+   - :guilabel:`Standard Pack`: 100 credits
+   - :guilabel:`Advanced Pack`: 500 credits
+   - :guilabel:`Expert Pack`: 1,000 credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
```

Comment:
I believe the packs.png image should be used here, as it provides a perfect visual representation of what's being discussed here